lib/rouge.rb

```diff
@@ -47,11 +47,9 @@ def load_file(path)
     #
     # @api private
     def load_lexers
-      # The trailing slash is necessary to avoid lexers being loaded multiple
-      # times by `Lexers.load_lexer`
-      lexer_dir = File.join(LIB_DIR, "rouge/lexers/")
-      Dir.glob(File.join(lexer_dir, '*.rb')).each do |f|
-        Lexers.load_lexer(f.sub(lexer_dir, ''))
+      lexer_dir = Pathname.new(LIB_DIR) / "rouge/lexers"
```

Comment:
**note:** The `/` operator is an overloaded operator that is defined in the `Pathname` class, it is similar to `File.join` in that it concatenates the different parts of a file path and automatically converts the slashes to the appropriate character for the current operating system, ie. cross-platform compatible.